src/viewer/CMakeLists.txt

```diff
@@ -11,9 +11,9 @@
 
 ADD_SUBDIRECTORY(rpc)
 ADD_SUBDIRECTORY(core)
+ADD_SUBDIRECTORY(proxy)
```

Comment:
This is sufficient to fix the build. I can run `exe/cli` but it hangs on `--help`

@cyrush any ideas?

Reply:
Here is a backtrace. Seems hung up waiting for a client to connect maybe?

```
#0  0x00007ffff716a4c7 in __libc_accept (fd=5, addr=..., len=0x7ffffffe5c84) at ../sysdeps/unix/sysv/linux/accept.c:26
#1  0x00007ffff731300f in RemoteProcess::SingleThreadedAcceptSocket() () from /home/local/KHQ/ryan.krattiger/Projects/spack/visit/src/dev/build-linux-ubuntu20.04-x86_64_v3-wzoomra/install/usr/local/3.4.9/linux-x86_64/lib/libvisitcommon.so
#2  0x00007ffff7313f55 in RemoteProcess::AcceptSocket() () from /home/local/KHQ/ryan.krattiger/Projects/spack/visit/src/dev/build-linux-ubuntu20.04-x86_64_v3-wzoomra/install/usr/local/3.4.9/linux-x86_64/lib/libvisitcommon.so
#3  0x00007ffff7314280 in RemoteProcess::FinishMakingConnection(int, int) ()
   from /home/local/KHQ/ryan.krattiger/Projects/spack/visit/src/dev/build-linux-ubuntu20.04-x86_64_v3-wzoomra/install/usr/local/3.4.9/linux-x86_64/lib/libvisitcommon.so
#4  0x00007ffff73189bb in RemoteProcess::Open(MachineProfile const&, int, int, bool) ()
   from /home/local/KHQ/ryan.krattiger/Projects/spack/visit/src/dev/build-linux-ubuntu20.04-x86_64_v3-wzoomra/install/usr/local/3.4.9/linux-x86_64/lib/libvisitcommon.so
#5  0x00007ffff7fb2cb2 in ViewerProxy::Create(char const*, int*, char***) ()
   from /home/local/KHQ/ryan.krattiger/Projects/spack/visit/src/dev/build-linux-ubuntu20.04-x86_64_v3-wzoomra/install/usr/local/3.4.9/linux-x86_64/lib/libviewerproxy.so
#6  0x00005555555e8c42 in visit_Launch ()
#7  0x00007ffff7901126 in cfunction_call (func=0x7ffff6adf920, args=<optimized out>, kwargs=<optimized out>) at Objects/methodobject.c:553
#8  0x00007ffff78abf87 in _PyObject_MakeTpCall (tstate=0x7ffff7cf1ad8 <_PyRuntime+166328>, callable=0x7ffff6adf920, args=<optimized out>, nargs=0, keywords=0x0) at Objects/call.c:214
#9  0x00007ffff784ba1d in _PyEval_EvalFrameDefault (tstate=<optimized out>, frame=<optimized out>, throwflag=<optimized out>) at Python/ceval.c:7307
#10 0x00007ffff79bc285 in _PyEval_EvalFrame (throwflag=0, frame=0x7ffff7723020, tstate=0x7ffff7cf1ad8 <_PyRuntime+166328>) at ./Include/internal/pycore_ceval.h:73
#11 _PyEval_Vector (args=0x0, argcount=0, kwnames=0x0, locals=0x7ffff6adba00, func=0x7ffff6ab60c0, tstate=0x7ffff7cf1ad8 <_PyRuntime+166328>) at Python/ceval.c:6434
#12 PyEval_EvalCode (co=co@entry=0x7ffff62295c0, globals=globals@entry=0x7ffff6adba00, locals=locals@entry=0x7ffff6adba00) at Python/ceval.c:1148
#13 0x00007ffff7a08c34 in run_eval_code_obj (locals=0x7ffff6adba00, globals=0x7ffff6adba00, co=0x7ffff62295c0, tstate=0x7ffff7cf1ad8 <_PyRuntime+166328>) at Python/pythonrun.c:1741
#14 run_mod (mod=mod@entry=0x55555571a2f0, filename=filename@entry=0x7ffff7cce630 <_PyRuntime+21776>, globals=globals@entry=0x7ffff6adba00, locals=locals@entry=0x7ffff6adba00, flags=flags@entry=0x0, arena=arena@entry=0x7ffff69fb0f0)
    at Python/pythonrun.c:1762
#15 0x00007ffff7a0abbe in PyRun_StringFlags (str=str@entry=0x5555556539c3 "visit.Launch()", start=start@entry=257, globals=0x7ffff6adba00, locals=0x7ffff6adba00, flags=flags@entry=0x0) at Python/pythonrun.c:1632
#16 0x00007ffff7a0ac2d in PyRun_SimpleStringFlags (command=0x5555556539c3 "visit.Launch()", flags=0x0) at Python/pythonrun.c:487
#17 0x00005555555870fe in main ()
```